WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -10,20 +10,24 @@ struct ItemListView: View {
 
     @State private var lastScrollPosition: CGFloat = 0
     @State private var showSimpleProductsModal: Bool = false
+    var itemListState: ItemListState {
```

Comment:
nit: this could be private?
```suggestion
    private var itemListState: ItemListState {
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/14727/commits/2a94b520944a91a5db51ba2ad6c711aedc9a6de3.